src/main/java/com/flightstats/hub/metrics/StatsdReporter.java

```diff
@@ -63,6 +65,10 @@ public void time(String name, long start, String... tags) {
         reportWithBothClients(statsDClient -> statsDClient.time(name, System.currentTimeMillis() - start, tags));
     }
 
+    public void timeTest(String name, long start, String... tags) {
```

Comment:
As mentioned in the other class, can this be deleted?

Reply:
done.